### DISCUSSION TITLE
Place for comments in Xml script file

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#206] [Ideas]
Hello,

It would be nice if there would be some line at the top of the script files to be able to add some comments on for example what the script does and what options are being used (similar to other programming languages). Or some other place / method to add comments would be great to, As we are doing different options with different settings it can get quickly confusing which version does what.

TXS

Cheers
Jean

**Comment by tmayerguerr** [comment]
Hi Jean,

Your wish is our command. We had the same idea. With our new [release](https://github.com/groops-devs/groops/releases/tag/2023-08-25) it is possible to add multi-lines comments  (add comments in the context menu) anywhere in the config file.

Cheers
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten,

:-) 

Great, that will make it a lot easier to track versions 
Cheers
Jean
